apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -160,35 +151,20 @@ export class FilesStorageUC {
 		await this.authorizationService.checkPermissionByReferences(userId, allowedType, parentId, context);
 	}
 
-	private checkFileName(entity: FileRecord, params: DownloadFileParams): void | NotFoundException {
-		if (entity.name !== params.fileName) {
-			this.logger.warn(`could not find file with id: ${entity.id} by filename`);
-			throw new NotFoundException(ErrorType.FILE_NOT_FOUND);
-		}
-	}
-
-	private checkScanStatus(entity: FileRecord): void | NotAcceptableException {
-		if (entity.securityCheck.status === ScanStatus.BLOCKED) {
-			this.logger.warn(`file is blocked with id: ${entity.id}`);
-			throw new NotAcceptableException(ErrorType.FILE_IS_BLOCKED);
-		}
-	}
-
 	public async download(userId: EntityId, params: DownloadFileParams) {
-		const entity = await this.fileRecordRepo.findOneById(params.fileRecordId);
+		const singleFileParams = { fileRecordId: params.fileRecordId };
```

Comment:
Sollte man hier const singleFileParams: SingleFileParmas = hinschreiben um den typ schon vor dem input als Parameter zu forcieren? Eigentlich ist das was dort passiert ein mapping. von DownloadFileParams auf SingleFileParams. 
Das beinhaltet (momentan) nicht viel, aber mappings sollten in mapper ..wer weiß was die Zukunft an Änderungen bringt. 

Bzg. meines Kommentars oben bei dem anderen Todo wo es um mapping von FileRecord auf x geht. Jetzt wo ich das hier noch mal gesehen haben ...sollte von helper zu mapper verschoben werden.

Reply:
Mapper wurde hinzugefügt